docs/source/en/using-diffusers/write_own_pipeline.mdx

```diff
@@ -10,27 +10,34 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-# Write your own inference pipeline
+# Understanding models and schedulers
 
 [[open-in-colab]]
 
-🧨 Diffusers allows you to freely exchange models and schedulers in the pipeline to create your own custom pipeline for inference. This guide will show you how to create a custom Stable Diffusion pipeline for inference with the [`LMSDiscreteScheduler`] instead of the default [`PNDMScheduler`].
+A diffusion model consists of two main components, the model and the scheduler. The model's primary function is to denoise noisy input samples. The scheduler handles all the details of the denoising process, such as the number of denoising steps and the amount of noise to add. Depending on the context, the scheduler does slightly different things:
```

Comment:
This can be a bit confusing as the "model" contains other models. We refer to them later as both "components" and "models". Not sure if attempting to clarify helps.

```suggestion
A diffusion model consists of two main components, the model itself and the scheduler. This is a very high-level conceptual view, as the model contains several components that are themselves pre-trained ML models, as we'll see later. From this view, the model's primary function is to denoise noisy input samples. The scheduler handles all the details of the denoising process, such as the number of denoising steps and the amount of noise to add. Depending on the context, the scheduler does slightly different things:
```